syz-fuzzer/proc.go

```diff
@@ -276,10 +276,18 @@ func (proc *Proc) enqueueCallTriage(p *prog.Prog, flags ProgTypes, callIndex int
 	})
 }
 
-func (proc *Proc) executeRaw(opts *ipc.ExecOpts, p *prog.Prog, stat Stat) *ipc.ProgInfo {
+func (proc *Proc) executeRaw(opts *ipc.ExecOpts, p *prog.Prog, stat Stat, enableDF Stat, disableDF Stat) *ipc.ProgInfo {
```

Comment:
I think you don't need to pass `enableDF` and `disableDF` here, as they are always the same (unlike `stat`)